components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -0,0 +1,336 @@
+				end
+				table.insert(maps, '&nbsp;• ' ..
+					tostring(CustomLeague:_createNoWrappingSpan(
+						Page.makeInternalLink(args['map' .. index], args['map' .. index] .. mapmode)
+					))
+				)
+				index = index + 1
+			end
+
+			table.insert(widgets, Title{name = 'Maps'})
+			table.insert(widgets, Center{content = maps})
+		end
+
+		if not String.isEmpty(args.team1) then
+			local teams = {Page.makeInternalLink(args.team1)}
+			local index  = 2
+
+			while not String.isEmpty(args['team' .. index]) do
+				table.insert(teams, '&nbsp;• ' ..
+					tostring(CustomLeague:_createNoWrappingSpan(
+						Page.makeInternalLink(args['team' .. index])
+					))
+				)
+				index = index + 1
+			end
+
+			table.insert(widgets, Title{name = 'Teams'})
+			table.insert(widgets, Center{content = teams})
+		end
+	elseif id == 'prizepool' then
+		return {
+			Cell{
+				name = 'Prize pool',
+				content = {CustomLeague:_createPrizepool(args)}
+			}
+		}
+	elseif id == 'liquipediatier' then
+		return {
+			Cell{
+				name = 'Tier',
+				content = {CustomLeague:_createTier(args)}
+			}
+		}
+	elseif id == 'organizer' then
+		local sponsors = mw.text.split(args.sponsors, ',', true)
+		table.insert(widgets, Cell{
+			name = 'Sponsor(s)',
+			content = table.concat(sponsors, '&nbsp;•')
+		})
+	end
+
+	return widgets
+end
+
+
+function CustomLeague:getWikiCategories(args)
+	local categories = {}
+
+	if not (String.isEmpty(args.individual) and String.isEmpty(args.player_number)) then
+		table.insert(categories, 'Individual Tournaments')
+	end
+
+	if String.isEmpty(args.game) then
+		table.insert(categories, 'Tournaments without game version')
+	else
+		table.insert(categories, GameLookup.getName({args.game}) .. (args.beta and ' Beta' or '') .. 'Competitions')
+	end
+
+	return categories
+end
+
+function CustomLeague:_createTier(args)
+	local cell = {name = 'Liquipedia Tier'}
+
+	local content = ''
+
+	local tier = args.liquipediatier
+
+	if String.isEmpty(tier) then
+		return cell
+	end
+
+	local tierDisplay = Template.safeExpand(mw.getCurrentFrame(),
+										'TierDisplay/link',
+										{ tier, GameLookup.getName({args.game})})
+	local type = args.liquipediatiertype
+
+	if not String.isEmpty(type) then
+		local typeDisplay = Template.safeExpand(mw.getCurrentFrame(), 'TierDisplay/link', type)
+		content = content .. typeDisplay .. ' (' .. tierDisplay .. ')'
+	else
+		content = content .. tierDisplay
+	end
+
+	cell.content = {content}
+	return cell
+end
+
+function CustomLeague:_createPrizepool(args)
+	if String.isEmpty(args.prizepool) and
+		String.isEmpty(args.prizepoolusd) then
+			return nil
+	end
+
+	local date
+	if not String.isEmpty(args.currency_rate) then
+		date = args.currency_date
+	else
+		date = args.edate or args.date
+	end
+
+	local content = PrizePool._get({
+			prizepool = args.prizepool,
+			prizepoolusd = args.prizepoolusd,
+			currency = args.localcurrency,
+			rate = args.currency_rate,
+			date = date
+		}
+	)
+
```

Comment:
```suggestion

Variables.varDefine('tournament_prizepoolusd', Variables.varDefault('tournament_prizepool_usd'))

```
this or set that var in https://liquipedia.net/commons/Module:Prize_pool_currency